pup-0002.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP proposes the use of Google style docstrings for Pulp 3+.
+
+## Motivation
+
+Most current Pulp developers find Google Style docstrings to be more readable than the default
+Sphinx style docstrings.
+
+## Details
+
+Upon acceptance, these changes will be documented in the Pulp style guide.
```

Comment:
I think this should get moved to a new section just above `Drawbacks` which would be called `Putting It Into Practice` which would identify several things:
* That this style would be documented in the style guide
* That the docs will be built with Napoleon as a plugin which supports this style and standard Sphinx style.